exercises/macros/.meta/description.md

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I feel like this extra line could go, unless it is very important to have two newlines after the title